reproducibility_project/bin/clean_by_engine.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+echo Removing jobs from all engines but $1
+for ENGINE in cassandra mcccs gomc gromacs hoomd lammps-VU lammps-UD
+do
+    if [[ $ENGINE != $1 ]]
+    then
+        read -p "Delete $ENGINE job directories?[y/N] " yn
+        case $yn in
+            [Yy]* ) signac rm $(signac find engine $ENGINE); echo Removed jobs for $ENGINE;;
```

Comment:
What exactly does the `[Yy]*` mean here? the [Yy] im assuming is a regex pattern for lowercase or capital `Y`.

What does the * mean though?

I understand that the `* )` means the `else` case essentially.

Reply:
> What exactly does the [Yy]* mean here?

Any input in [y, Y, yes, Yes, yep, Yankovic, yellow, etc] (anything starting with y or Y) will result in the directories being deleted.

> I understand that the * ) means the else case essentially.

Yep! Even if the user accidentally presses enter, nothing will be deleted.

Reply:
Perfect!